src/dash-adapter.js

```diff
@@ -398,7 +398,7 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
     //Need to call this again cause we are uninstalling the VTTCue polyfill to avoid collisions with other libs
     shaka.polyfill.installAll();
     this._shaka = new shaka.Player();
-    //render text tracks to our own container
+    // This will force the player to use shaka UITextDisplayer plugin to render text tracks.
```

Comment:
is the comment relevant ?

Reply:
yes, because this is not clear what this line does without a comment
event when you look at the dash DOCS is not clear what it does in our context